update facebook docs with supported attachments (#5693)
RasaHQ/rasa: docs/user-guide/connectors/facebook-messenger.rst

``http://localhost:5005/webhooks/facebook/webhook``, replacing
the host and port with the appropriate values. This is the URL
you should add in the configuration of the webhook.

> Supported response attachments
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Supported response attachments
------------------------------
```

> In addition to typical text, image, button, and custom responses, the Facebook Messenger channel supports the following additional response template attachments:

Reviewer: ```suggestion
In addition to typical text, image, and custom responses, the Facebook Messenger channel supports the following additional response template attachments:
```
because we also reference how buttons are different here

> * `Buttons <https://developers.facebook.com/docs/messenger-platform/send-messages/buttons>`_: sends a text message with up to three attached buttons
> 
> .. code-block:: yaml
> 
>    utter_fb_buttons_example:
>       - text: Hello World!
>         buttons:
>           - title: Button Title
>             payload: /example_intent
>           - title: Button Title
>             payload: /example_intent
>           - title: Button title over 20 characters # this will be truncated to: "Button title over..."
>             payload: /example_intent
> 
> .. warning::
> 
>    Facebook API limits the amount of buttons you can sent in a message to 3.  If more than 3 buttons are provided in a message, Rasa will ignore all provided buttons.

Reviewer: ```suggestion
* `Buttons <https://developers.facebook.com/docs/messenger-platform/send-messages/buttons>`_: are structured the same as other Rasa buttons. Facebook API limits the amount of buttons you can sent in a message to 3. If more than 3 buttons are provided in a message, Rasa will ignore all provided buttons.
```

* `Quick Replies <https://developers.facebook.com/docs/messenger-platform/send-messages/quick-replies>`_: provide a way to present a set of up to 13 buttons in-conversation that contain a title and optional image, and appear prominently above the composer. You can also use quick replies to request a person's email address or phone number.

.. code-block:: yaml

   utter_fb_quick_reply_example:
      - text: Hello World!
        quick_replies:
          - title: Text quick reply
            payload: /example_intent
          - title: Image quick reply
            payload: /example_intent
            image_url: http://example.com/img/red.png
          # below are Facebook provided quick replies
          # the title and payload will be filled
          # with the user's information from their profile
          - content_type: user_email
            title:
            payload:
          - content_type: user_phone_number
            title:
            payload:

.. note::

   Both Quick Reply and Button titles in Facebook Messenger have a character limit of 20.  Titles longer than 20 characters will be truncated.

> * `Generic Template Elements <https://developers.facebook.com/docs/messenger-platform/send-messages/template/generic>`_: provide a way to create a horizontally scrollable list up to 10 content elements that integrate buttons, images, and more alongside text a single message.

Reviewer: ```suggestion
* `Elements <https://developers.facebook.com/docs/messenger-platform/send-messages/template/generic>`_: provide a way to create a horizontally scrollable list up to 10 content elements that integrate buttons, images, and more alongside text a single message.
```
